src/Symfony/Component/DependencyInjection/Compiler/AutowireRequiredMethodsPass.php

```diff
@@ -49,6 +50,14 @@ protected function processValue($value, bool $isRoot = false)
             }
 
             while (true) {
+                if (method_exists($r, 'getAttributes') && $r->getAttributes(Required::class)) {
```

Comment:
let's use PHP_VERSION_ID, that will ease spot the line when PHP8 will be the minimum supported version (yes, that will happen one day :) )